.eslintrc.json

```diff
@@ -22,6 +22,7 @@
   "rules": {
     "prettier/prettier": "error",
     "comma-dangle": 0,
+    "@typescript-eslint/consistent-type-imports": "error",
```

Comment:
This is a very helpful rule. https://typescript-eslint.io/rules/consistent-type-imports/

I've always preferred the clarity of doing `import type` when importing types. It also prevents naming collisions that happen when a type and a component have the same name (like in the `Filter` component which takes data of type `Filter` for example). This rule makes the editor do it automatically. IMO that's how it should work by default